examples/e2e-test-dataflow/Readme.md

```diff
@@ -1,6 +1,6 @@
 # Running End-to-End test for Dataflow pipelines
```

Comment:
While you are at it could you also rename this file to README all capital please ? 

Thanks